simpeg_drivers/electricals/direct_current/two_dimensions/__init__.py

```diff
@@ -15,5 +15,6 @@
 #
 # ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-
 from .params import DirectCurrent2DParams
+
+__all__ = ["DirectCurrent2DParams"]
```

Comment:
note: we should actually do that in every `__init__.py`
(and not ignore flake8)

I did not know about `__all__` when we initiated the work on geoh5py